backend/models/travel-model.js

```diff
@@ -0,0 +1,36 @@
+import mongoose from 'mongoose';
+
+const travelSchema = new mongoose.Schema({
+  distance: {
+    type: Number,
+    required: [true, "Distance is required"],
+    min: [0, "Distance cannot be negative"],
+  },
+  wayOfTravel: {
+    type: String,
+    enum: ["driving", "walking", "bicycling", "transit"],
+    required: [true, "Way of travel is required"],
+  },
+  date: {
+    type: Date,
+    required: [true, "Date is required"],
+  },
+  startLocation: {
+    type: String,
+    required: [true, "Start location is required"],
+  },
+  endLocation: {
+    type: String,
+    required: [true, "End location is required"],
+  },
```

Comment:
Are this 2 values to store the history of travels in the Dashboard ?

Reply:
Its for storing the start and end of a route - so we can use it for multiple things:
 if we want to use this data later for 
 calculation, statistics or maybe show popular routes etc.
 
Now we store it on each travelroute the user preforms - so yes! this will let us show detailed info for previous routes in our dash.